test/e2e/sourcery.js

```diff
@@ -1,62 +1,108 @@
-var conf = require('../../nightwatch.conf.js');
+//var conf = require('../../nightwatch.conf.js');
+const c = require('../../libs/constants');
+const common= require('../../obj/common');
+const login = require('../../obj/login');
+const timeLogging = require('../../obj/timeLogging');
+const tasks = require('../../obj/tasks');
+const createTask = require('../../obj/createTask');
 
 module.exports = {
     'Login to sourcebooks': function (browser) {
-        browser
-        .url(browser.launchUrl)
-        .waitForElementVisible('h1'); // wait for the Login title
-        
-        //Click to expand select user dropdown
-        browser.element('css selector', '#react-select-2--value', function(result) {
-            if(result.status != -1) { 
-                browser.click('#react-select-2--value');
-            }
-        });
-        
-        //Select from expanded droprown
-        browser.element('css selector', '[aria-label="Linas Steponavicius"]', function(result) {
-            if(result.status != -1) { 
-                browser.click('css selector', '[aria-label="Linas Steponavicius"]');
-            }
-        });
 
-        //Assert value is selected
-        browser.assert.containsText('#react-select-2--value-item', 'Linas Steponavicius');
-        
-        //Click to expand select role dropdown
-        browser.element('css selector', '#react-select-3--value', function(result) {
-            if(result.status != -1) { 
-                browser.click('css selector', '#react-select-3--value');
-            }
-        });
-        
-        //Select from expanded droprown
-        browser.element('css selector', '[aria-label="Admin"]', function(result) {
-            if(result.status != -1) { 
-                browser.click('css selector', '[aria-label="Admin"]');
-            }
-        });
+        let user = "Linas Steponavicius";
+        let role = "Admin";
 
-        //Assert value is selected
-        browser.assert.containsText('#react-select-3--value-item', 'Admin');
-        
-        //Click submit button
-        browser.element('css selector', '[type="submit"]', function(result) {
-            if(result.status != -1) {
+        browser
+        .url(browser.launchUrl)
+        .waitForElementVisible(common.pageTitle)
+        .isVisible(login.userSelect, function(result){  //Click to expand the user dropdown
+            if(result.status === c.ELEMENT_FOUND)
+                browser.click(login.userSelect);
+        })
```

Comment:
Hint for future improvements (out of scope for this PR), this does not handle errors, if select is not found nothing is clicked, no error message is printed out and failed step is not in the right place in case of button not found.